mu4/palette/qml/MuseScore/Palette/PaletteGrid.qml

```diff
@@ -80,7 +79,7 @@ Canvas {
         }
         const offY = offsetY % cellHeight;
         const nrows = Math.ceil((height - offY) / cellHeight) + (offY ? 1 : 0);
-        for (var i = 1; i < nrows; ++i) {
+        for (i = 1; i < nrows; ++i) {
```

Comment:
does it work now?

Reply:
Dude, this is JS :) This line generated a warning because the variable i was already declared

Reply:
Better find where `i` is globally declared and make it local